lisa/wlgen/rta.py

```diff
@@ -63,13 +65,14 @@ def _early_init(self, target, name, res_dir, json_file):
             json_file = '{}.json'.format(self.name)
 
         self.local_json = ArtifactPath.join(self.res_dir, json_file)
-        self.remote_json = self.target.path.join(self.run_dir, json_file)
+        remote_json_file = '{}_{}'.format(uuid.uuid4(), json_file)
```

Comment:
Might want to swap the order? If we end up with several in a directory they'll be sorted together.

Reply:
If we swap the order we will get `foo.json_XXXXXX`, and since the file name can come from the user I figured it would be better to avoid assumptions on the (multiple) extensions the file could have.
Once https://github.com/ARM-software/lisa/pull/958 is merged we could actually drop that commit and only keep the `shlex.quote()` one, since the folder containing it will be unique.